doc/jvm/xgboost4j_spark_tutorial.rst

```diff
@@ -153,6 +153,54 @@ Now, we have a DataFrame containing only two columns, "features" which contains
 "sepal length", "sepal width", "petal length" and "petal width" and "classIndex" which has Double-typed
 labels. A DataFrame like this (containing vector-represented features and numeric labels) can be fed to XGBoost4J-Spark's training engine directly.
 
+Dealing with missing values (NaN / Null)
```

Comment:
Let's make `dealing with missing values` as a subsection of `Transform Raw Iris Dataset` instead a peer section

Reply:
@CodingCat done, is part of updated pull request (hopefully final pull req.).